crates/net/discv4/src/lib.rs

```diff
@@ -56,6 +56,7 @@ use std::{
     task::{ready, Context, Poll},
     time::{Duration, Instant, SystemTime, UNIX_EPOCH},
 };
+use itertools::Itertools;
```

Comment:
I think this is ok given we already have it in the workspace

Reply:
yes, It's in the workspace